react-website/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:16.17.1-buster-slim
+
+RUN mkdir -p /home/app
+
+WORKDIR /home/app
+
+COPY . /home/app
+
+RUN mkdir -p /home/app/themes-dist
```

Comment:
It is not necessary to create the directories here. The bind mount should be able to create these directories for us.